### PR TITLE
Adds a new abandoned area at science

### PR DESCRIPTION
**Pull request by monster860**
Because you can ALWAYS use another abandoned area.

:cl: monster860
rscadd: Adds a new abandoned area to the station at science
tweak: Changes the icon_regular_floor behavior to not change the value if not necessary.
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
Where is this compared to the rest of Science/the station?

**Comment by monster860** [comment]
Right across from the test lab

**Comment by QuinnAggeler** [comment]
You might want to provide a bigger screenshot so it's more apparent where this is

**Comment by Dinarzad** [comment]
I like it, 's a neat spot for Science to set up Telescience or some other contraptions, but I'd suggest making it a bit smaller. There's not much they USUALLY build that needs quite that much room.
About where the Air and Fire Alarm are would be fine IMO, with the Reinforced walls on the same line where you have the Light switch at in the current image.

**Comment by Fox-McCloud** [comment]
On the chance that we institute firing pins, it's pretty vital that there be a test firing range (to make use of the firing range pins)---this area is prime ground for that, and, unfortunately, space in science comes at a premium---and there's not really any other room.

**Comment by Spacemanspark** [comment]
I hate firing pins.

**Comment by monster860** [comment]
@Fox-McCloud We can always move xenobio maintenance a few tiles over.

**Comment by ProperPants** [comment]
@monster860 Are those two little windows in the north end of the room looking into the maint tunnel? If so, that might be too easy a way for antags to enter science.

**Comment by monster860** [comment]
@ProperPants that was an oversight, fixed.

File: code/game/turfs/simulated/floor.dm [hunk 1]
```diff
@@ -33,9 +33,7 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 
 /turf/simulated/floor/New()
 	..()
-	if(icon_state in icons_to_ignore_at_floor_init) //so damaged/burned tiles or plating icons aren't saved as the default
-		icon_regular_floor = "floor"
-	else
+	if(!(icon_state in icons_to_ignore_at_floor_init)) //so damaged/burned tiles or plating icons aren't saved as the default
```
Comment (tigercat2000) [inline code/game/turfs/simulated/floor.dm]: this is entirely outside the bounds for this PR to be atomic

Comment (monster860) [inline code/game/turfs/simulated/floor.dm]: Should I do this in another PR?

Comment (tigercat2000) [inline code/game/turfs/simulated/floor.dm]: yes

**Comment by Kyep** [comment]
This could always be converted to a firing range later, if we need one.
I don't think "we might need a firing range later" is a good reason not to add this.

**Comment by taukausanake** [comment]
I can see the room being used for telescience actually. I nice, small room that needs work. Or anything else. Maybe relocate the EXPERIMentor to it so people stop dying so commonly. Or reinforce it and use it for Test Lab because they will explode eventually. Might I also say it might be enough room for a small supermatter engine? The possibilities are endless!

**Comment by tigercat2000** [comment]
@taukausanake 
> I can see the room being used for telescience actually. 

yeah that's the point against having it

**Comment by ProperPants** [comment]
@tigercat2000
But literally any room can be used for telescience. The Test Lab or the RD's office often end up being that room.

**Comment by Spacemanspark** [comment]
Honestly we should just gut out telescience from the code entirely, the only thing it does is cause issues.
That said, I'd be down to have a room like this in science.

**Comment by tigercat2000** [comment]
@ProperPants i realize now that i wasn't entirely clear
i don't really care one way or another for this, mapping is awful and it can be overwritten later anyways
but a few others realized the exact same point of "used for telescience" and use it as a negative point not a positive point

**Comment by taukausanake** [comment]
Well if people don't like telescience then you really should get rid of it. It's such an issue then it should be treated like one.

Otherwise, I was thinking this room needed a camera but then I noticed it doesn't have power hooked up. Unless someone with gloves gets in there it might not be used so much. Also, does the door belong to the room or the hall? It should belong to the room kind of like the Assembly Line room

**Comment by TheDZD** [comment]
I don't really see the point of giving them an abandoned space, considering RD already has relatively large rooms with movable/removable machinery/structures, as well as the entire research outpost to play around with.

**Comment by ProperPants** [comment]
It's kinda tedious to ferry back and forth between to station and outpost to keep getting what you need.

**Comment by taukausanake** [comment]
I still think the Science Outpost should be nuked. If I could busy myself with mapping and could use map merge then I might try blowing it up a bit. I don't know though

**Comment by Kevak** [comment]
I like the sci outpost. It just needs to be refurbished. Set up xenobot there. Maybe finish xenoarch.

**Comment by Fox-McCloud** [comment]
Closing for reasons stated.
